roles/openshift_persistent_volumes/tasks/main.yml

```diff
@@ -23,7 +23,7 @@
     --config={{ mktemp.stdout }}/admin.kubeconfig
   register: pv_create_output
   when: persistent_volumes | length > 0
-  failed_when: ('already exists' not in pv_create_output.stderr if pv_create_output.stderr else False) or ('created' not in pv_create_output.stdout if pv_create_output.stdout else False)
+  failed_when: ('already exists' not in (pv_create_output.stderr | default(''))) and ('created' not in (pv_create_output.stdout | default('')))
```

Comment:
This seems like it'd yield false failures, I wonder if the wrapper script broke stdout / stderr splitting.